packages/next-dashboard/src/components/charts/Chart.js

```diff
@@ -52,7 +52,7 @@ const Chart = ({
   title,
   data,
   chartLineColor,
-  loading = false,
+  loading,
```

Comment:
Should this not default to `false`? that seems like a sensible default

Reply:
It was an unsaved file, meant to be part of the initial commit. IIRC I changed this when I realized that flow didn't complain about keeping undefined things as false-y, and I got lazy with future defaults and didn't make the default explicitly boolean in other places, so I changed this back to be consistent, forgot to save, and it got included in this commit. Otherwise, the solution is to go through the codebase to find all the other places where we have boolean values defaulting to undefined and make them explicitly default to false instead

Reply:
I really prefer keeping things sanely typed and setting an explicit boolean default value for a boolean argument, rather than relying on sloppy typing. To me consistency isn't a great argument in favour of keeping things worse; I'd prefer gradual improvement, fixing it in places as one stumbles upon them.

Reply:
Sure but I'm not reopening this branch in my editor to fix this right now, I just made this PR so I could continue working on other stuff when I noticed the unsaved files from the last PR where, if this reverted change was there, you wouldn't even have noticed this :D

Reply:
I guess :p